.changeset/sweet-aliens-crash.md

```diff
@@ -0,0 +1,13 @@
+---
+"@aptos-labs/derived-wallet-ethereum": minor
+"@aptos-labs/derived-wallet-solana": minor
+"@aptos-labs/derived-wallet-base": minor
+"@aptos-labs/wallet-adapter-core": minor
+"@aptos-labs/cross-chain-core": minor
+"@aptos-labs/wallet-adapter-ant-design": minor
+"@aptos-labs/wallet-adapter-mui-design": minor
+"@aptos-labs/wallet-adapter-react": minor
```

Comment:
All those packages should be major. it will break for dapps that still use previous ts-sdk versions.
"@aptos-labs/wallet-adapter-ant-design"
"@aptos-labs/wallet-adapter-mui-design"
"@aptos-labs/wallet-adapter-react"
"@aptos-labs/wallet-adapter-vue"